packages/core/src/logger/Logger.spec.ts

```diff
@@ -42,13 +42,17 @@ const getLogLevelName = (logLevel: LogLevel): string => {
 };
 
 describe('Logger', () => {
+  const mockedConsole = jest.spyOn(console, 'log');
+
+  afterEach(() => jest.resetAllMocks());
+
   describe('level config', () => {
     it('default level is NOTICE', () => {
       const logger = new Logger();
       expect(logger.logLevel).toEqual(LogLevel.NOTICE);
     });
 
-    it('level can be set via contructor', () => {
+    it('level can be set via contractor', () => {
```

Comment:
great changes, just not sure about this specific wording

Reply:
😄 so, it is auto spelling correction in WebStorm)